lib/pytis/data/test.py

```
@@ -204,6 +209,15 @@ def test_equality(self):
 class TestFloat(_TestType):
     _type = pd.Float
 
+    def setUp():
+        import locale
+        self._orig_locale = locale.getlocale(locale.LC_ALL)
```

Ruff (0.12.2)

Undefined name `self`

(F821)